src/ui/OGMessageItemBody/index.tsx

```diff
@@ -37,8 +37,15 @@ export default function OGMessageItemBody({
 }: Props): ReactElement {
   const imageRef = useRef<HTMLDivElement>(null);
   const { stringSet } = useContext(LocalizationContext);
+
   const openOGUrl = (): void => {
-    if (message?.ogMetaData?.url) window.open(message?.ogMetaData?.url);
+    let url = message?.ogMetaData?.url;
+    if (url) {
+      if (!url.startsWith('http')) {
+        url = 'https://' + url;
```

Comment:
@bang9 is this proven to be safe?

Reply:
How about below?
```ts
  if (!url.startsWith('http://') && !url.startsWith('https://')) {
    url = 'https://' + url;
  }
```
I will make the same changes for react-native as well.

Reply:
Yes definitely!

Reply:
but what i meant is is adding https:// safe? is there any case that url would not work after adding https at the start?

Reply:
More precisely, if site doesn't support HTTPS, it won't open.
However, for sites that have implemented at least basic handling for OG meta tags, it is expected that HTTPS would be applied lol.